changelog.md

```diff
@@ -1,6 +1,7 @@
 # changelog
 
-## 1.1.0
+## 1.1.1
```

Comment:
Suggest we keep v1.1.1 distinct from v1.1.0, since the latter artifact technically *was* released (unlike 1.0.1 → 1.0.2).

Reply:
THank you, yes - amended